examples/ERP/plot_classify_MEG_mdm.py

```diff
@@ -62,7 +71,7 @@
 n_components = 3  # pick some components
 
 # Define a monte-carlo cross-validation generator (reduce variance):
```

Comment:
it's a plain kfold you use here.

Reply:
thats probably some older comment that was left from using the sufflesplit ... thanks for catching it.